src/Components/Notifications/NotificationsList.tsx

```diff
@@ -320,9 +323,9 @@ export default function NotificationsList({
 
   useEffect(() => {
     setIsLoading(true);
-    dispatch(
-      getNotifications({ offset, event: eventFilter, medium_sent: "SYSTEM" })
-    )
+    request(routes.getNotifications, {
+      query: { offset, event: eventFilter, medium_set: "SYSTEM" },
+    })
       .then((res: any) => {
```

Comment:
The `any` type can be avoided since `useQuery/request` is typesafe!

```suggestion
      .then((res) => {
```